tests/integration/dev.bats

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+	teardown_busybox
+	setup_busybox
+}
+
+function teardown() {
+	teardown_busybox
+}
+
+@test "runc run [redundant default dev]" {
```

Comment:
Could you add a test for ptmx as well? (It had a slightly code path than tty)

e.g. https://github.com/opencontainers/runc/pull/2451/files#diff-b890de221ad09cb84cc3c69482f2da36R235

Reply:
Sure, I'll add the test for ptmx too.